packages/manager/apps/dedicated/client/app/account/identity-documents/user-identity-documents.controller.js

```diff
@@ -2,13 +2,25 @@ import {
   USER_TYPE,
   PROOF_TYPE,
   TRACKING_TASK_TAG,
+  TRACKING_VARIABLES,
   LEGAL_LINK1,
   LEGAL_LINK2,
   LEGAL_LINK3,
   KYC_STATUS,
   DOCUMENTS_MATRIX,
 } from './user-identity-documents.constant';
 
+const replaceTrackingParams = (hit, params) => {
+  if (!params) return hit;
+  let formatted = hit;
+  params.forEach((param) => {
+    const { key, value } = Object.entries(param);
```

Comment:
also what is the type of params supposed to be ? an array ? if it's an object I think it would be more like

`
Object.entries(params).forEach(([key, value]) => {
// do your stuff here
})
`

Reply:
see example here https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Global_Objects/Object/entries#parcourir_un_objet